src/components/Chromosome.tsx

```diff
@@ -18,28 +17,39 @@ function Chromosome({
 }: ChromProps): JSX.Element {
   // Render band within the bandosome
   //console.log(props.selectedLocations);
+  let y = 16;
```

Comment:
Maybe we should have a yaml file or something with these configurations.

Reply:
we can add this in my next PR